app/html/app.js

```diff
@@ -56,6 +59,29 @@ exports.create = function (api) {
       addError(ev.error || ev)
     })
 
+    ////// TODO - extract this to keep patch-lite isolated from electron
+    const { getCurrentWebContents, getCurrentWindow } = electron.remote
+    window.addEventListener('resize', () => {
+      var wc = getCurrentWebContents()
+      wc && wc.getZoomFactor((zf) => {
```

Comment:
@arj03 I've set it up so the settings takes objects - I've changed it to use this for name-spacing